docs/polymer-modulize-howto.md

```diff
@@ -0,0 +1,15 @@
+## How to modulize and publish Polymer + elements
+
+1. Edit `dependency-map.json` and bump every `^3.0.0-pre.<N>` version to
+   `^3.0.0-pre.<N+1>`.
+
+2. Run `modulizer --clean --npm-version 3.0.0-pre.<N+1> --repo $(cat <modulizer dir>/docs/polymer-modulize-repos.txt | tr '\n' ' ') --delete-files types`. This should take 2-3 minutes.
```

Comment:
super nit: I'd say ` --delete-files types/` to make it clear you're trying to delete a types directory, and not some keyword that "types" represents.